optuna/study.py

```diff
@@ -386,7 +387,9 @@ def objective(trial):
             show_progress_bar=show_progress_bar,
         )
 
-    def ask(self) -> trial_module.Trial:
+    def ask(
+        self, fixed_distributions: Optional[Dict[str, BaseDistribution]] = None
```

Comment:
[Question] We sometimes call a dictionary of distributions `search_space` as can be seen in the arguments of [`Sampler.sample_relative`](https://github.com/optuna/optuna/blob/85f4d1c41baa00ea0c666fd6c1fc3924cca2d14e/optuna/samplers/_base.py#L75) . So this argument could be `fixed_search_space`.
I think you intentionally avoid the name because users may think that they cannot add parameters other than `fixed_search_space`. Actually, we can suggest the parameters which are not in the argument. Therefore, you chose `fixed_distributions` instead of `fixed_search_space`. Does it make sense?

```python
trial = study.ask({"x": UniformDistribution(1, 10)})

trial.suggest_float("y", 0, 1)  # Some users may think 
```

Reply:
Right; `search_space` might be rather confusing especially for ppl who are accustomed with the define-and-run style. In addition, we use the term of `distributions` in the argument of [`create_trial`](https://optuna.readthedocs.io/en/stable/reference/generated/optuna.trial.create_trial.html#optuna.trial.create_trial).

Reply:
Thank you for your answer. Now, I'm confident with the choice of `fixed_distribution`.